app/views/menu/index.html.erb

```diff
@@ -5,7 +5,7 @@
       <%= item.name %>
       <%= button_to "Add to Cart", cart_items_path(item: item), class: "btn btn-default" %>
       <div class="btn-group btn-group-xs" role="group">
-      <% item.categories.each do |item_category| %>
+      <% item.categories.uniq.each do |item_category| %>
```

Comment:
This is in case an item is tagged with the same category more than once, it will only display that category one time. This is probably something @mirjoy saw in the actual dev app, which is probably my fault since I may not have sanitized it correctly in my seed data. This is a good failsafe though, in case a user ever tried to tag it.